apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ast/Ast.kt

```diff
@@ -1,6 +1,6 @@
-package com.apollographql.apollo.compiler.ast
+package com.apollographql.apollo.compiler.backend.ast
 
-internal typealias ObjectTypeContainer = Map<CodeGenerationAst.TypeRef, CodeGenerationAst.ObjectType>
+import com.apollographql.apollo.compiler.parser.introspection.IntrospectionSchema
 
 internal typealias CustomTypes = Map<String, CodeGenerationAst.CustomType>
 
```

Comment:
Can we add a top level Kdoc block to explain what the backend `Ast` is? I'd expect it to be very close to the Kotlin language but the `Operation` and `Fragment` below let me think it's not?

Reply:
I will think if we can remove AST and directly generate code from BackendIR

Reply:
Actually I will take this back. We do some `BackendIr lowering` here so we should keep it.